Gemfile

```diff
@@ -44,6 +44,9 @@ gem "fastly", "~> 2.5" # Client library for the Fastly acceleration system
 gem "feedjira", "~> 3.1" # A feed fetching and parsing library
 gem "field_test", "~> 0.3" # A/B testing
 gem "figaro", "~> 1.2" # Simple, Heroku-friendly Rails app configuration using ENV and a single YAML file
+gem "flipper", "~> 0.17.2" # Feature flipping / flags for Ruby
+gem "flipper-active_record", "~> 0.17.2" # Store Flipper flags in ActiveRecord
```

Comment:
I think FeatureFlags should be in the DB and then we can optimize them with Redis. The reason I think this is that I always try to design a system so that if Redis goes down we dont lose anything crucial. Redis in my opinion should be there to improve things but when possible not be the source of truth for anything. At least that is my opinion when Redis is used in our context. Obviously many people do use it as their main datastore and that works too!

Reply:
Thanks, makes perfect sense to me! Going back to redundancy conversations :)